<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>513</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto Medium</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Invoice Generator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 31, 31);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>424</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border :1px solid ;
border-top-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="heading">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>591</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>591</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>(date) &gt; (date) - (firstName lastName)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_day">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ph">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Public Holiday?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hours">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hours</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>151</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="3" column="0">
      <widget class="QCheckBox" name="cb_ph_w">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="cb_ph_sa">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="cb_ph_su">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="cb_ph_tu">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="cb_ph_m">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="cb_ph_th">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="cb_ph_f">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="prevButton">
    <property name="geometry">
     <rect>
      <x>388</x>
      <y>424</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border :1px solid ;
border-top-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>&lt; Prev</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>191</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_sun">
       <property name="font">
        <font>
         <family>Roboto Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sunday (date)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_mon">
       <property name="font">
        <font>
         <family>Roboto Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Monday (date)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_tues">
       <property name="font">
        <font>
         <family>Roboto Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tuesday (date)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_wed">
       <property name="font">
        <font>
         <family>Roboto Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wednesday (date)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_thurs">
       <property name="font">
        <font>
         <family>Roboto Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Thursday (date)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_fri">
       <property name="font">
        <font>
         <family>Roboto Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Friday (date)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_sat">
       <property name="font">
        <font>
         <family>Roboto Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Saturday (date)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>17</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_day_2">
       <property name="font">
        <font>
         <family>Roboto Medium</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kilometres</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="input_kms"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>431</x>
      <y>133</y>
      <width>131</width>
      <height>280</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="input_w_su"/>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLineEdit" name="input_w_m"/>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLineEdit" name="input_w_tu"/>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLineEdit" name="input_w_w"/>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLineEdit" name="input_w_th"/>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLineEdit" name="input_w_f"/>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLineEdit" name="input_w_sa"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="skipButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>424</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Medium</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border :1px solid ;
border-top-color: rgb(255, 255, 255);
border-bottom-color: rgb(255, 255, 255);
border-right-color: rgb(255, 255, 255);
border-left-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Skip &gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
